src/test/java/net/snowflake/client/jdbc/StatementLatestIT.java

```diff
@@ -225,4 +226,26 @@ public void testPreparedStatementLogging() throws SQLException {
       }
     }
   }
+
+  @Test // SNOW-647217
+  public void testSchemaWith255Characters() throws SQLException {
+    String schemaName = generateRandomString(255);
+    try (Connection con = getConnection()) {
+      try (Statement stmt = con.createStatement()) {
+        stmt.execute("create schema " + schemaName);
+        stmt.execute("use schema " + schemaName);
+        stmt.execute("drop schema " + schemaName);
+      }
+    }
+  }
+
+  private static String generateRandomString(int length) {
```

Comment:
`randomAlphaNumeric` is already present.

Reply:
changed, thanks